platform-sdk/docs/core/dynamic-address-book/DynamicAddressBook-PlatformDesign.md

```diff
@@ -0,0 +1,107 @@
+# Dynamic Address Book - Platform Design
+
+![Dynamic Address Book Platform Design](DynamicAddressBook-PlatformDesign.drawio.svg)
+
+## Network Connection Effects
+
+Assumption: That the network is fully connected, all nodes connect to each other.
+
+* Need to validate that this is true.
+* Huge gap in testing if not.
+
+### Add Node
+
+Existing nodes need to create or receive mutual TLS gossip connections with the added node.
+
+This change takes effect when a node is added in the `currentEffectiveRoster` for the `pendingConsensusRound`.
```

Comment:
I think the wording here can be more clear. The change takes effect when a roster with an added node becomes the `currentEffectiveRoster`.

Reply:
updating the wording to "This change takes effect when a roster with ___ becomes the currentlyEffectiveRoster...